### PR TITLE
[INLONG-2509][Feature][Audit][DockerCompose] Add support for Audit on Docker Compose

### PR DESCRIPTION
**Pull request by shink**
Fixes: #2509

### Motivation

The audit feature has been brought in #1738. Next we should add support for it on Docker Compose.

### Modifications

- `docker/docker-compose/docker-compose.yml`
- `docker/docker-compose/README.md`

### CODE REVIEW

File: docker/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -123,4 +136,20 @@ services:
...
-      - ./collect-data:/data/collect-data
\ No newline at end of file
+      - ./collect-data:/data/collect-data
+
+  audit:
+    image: inlong/audit:latest
+    container_name: audit
+    depends_on:
+      mysql:
+        condition: service_healthy
```
Comment (shink) [inline docker/docker-compose/docker-compose.yml:146]: @dockerzhang BTW, version 3 of Docker Compose no longer supports the `condition` form of `depends_on`, should we adapt it?

File: docker/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -33,6 +33,19 @@ services:
...
       - ./sql:/docker-entrypoint-initdb.d
 
+  pulsar:
+    image: apachepulsar/pulsar
+    container_name: pulsar
+    environment:
+      - PULSAR_MEM="-Xms256m -Xmx512m -XX:MaxDirectMemorySize=1g"
+    ports:
+      - "6650:6650"
+      - "8181:8080"
```
[the remark this answers is not in the document]
Comment (dockerzhang) [reply docker/docker-compose/docker-compose.yml:43]: the 8080 port for pulsar admin url is not used by audit ,we can remove this port mapping.

**Review by dockerzhang — APPROVED** [review]

File: docker/docker-compose/README.md [hunk 3]
```diff
@@ -9,10 +9,11 @@ Requirements:
...
 ### Deploy
 
-Manually copy SQL files from `inlong-manager/sql` to the `docker/compose/sql` directory.
+Manually copy SQL files from `inlong-manager/sql` and `inlong-audit/sql` to the `docker/docker-compose/sql` directory.
 
 ```shell
-cp inlong-manager/sql/apache_inlong_manager.sql  docker/docker-compose/sql
+cp inlong-manager/sql/apache_inlong_manager.sql docker/docker-compose/sql
+cp inlong-audit/sql/apache_inlong_audit.sql docker/docker-compose/sql
 ```
```
Comment (shink) [inline docker/docker-compose/README.md:17]: @dockerzhang It seems that `apache_inlong_manager.sql` is not in the `inlong-manager/sql` directory. Which path should be changed to?

Comment (dockerzhang) [reply docker/docker-compose/README.md:17]: after compiled, `apache_inlong_manager.sql` in `inlong-manager/sql/`,`apache_inlong_audit.sql` in `inlong-audit/sql/`

**Comment by dockerzhang** [comment]
@shink please update the related document on [office website](https://inlong.apache.org/docs/next/deployment/docker), the project repo is at `https://github.com/apache/incubator-inlong-website`.

**Comment by shink** [comment]
@dockerzhang OK.

**Review by healchow — APPROVED** [review]

**Pull request merged**